### ISSUE TITLE
Remove Bootstrap and replace by Tailwind

### ISSUE THREAD

**Issue by tiagostutz** [bancodobrasil/stop-analyzing-embed#49]
Currently we are using a bootstrap theme and classes. Remove the <link> from index.html, remove the theme.css and replace the Bootstrap styles in all HTML elements by Tailwind.

**Comment by giteshk11** [comment]
@divyaYK  is working on #30, so we can use complete this issue with that one.
What do you say? @tiagostutz

**Comment by tiagostutz** [comment]
> @divyaYK is working on #30, so she can complete this issue with along that one.
> What do you say? @tiagostutz

I offered @ishagupta1548 so she can start with an issue. I think they could go in parallel, by @divyaYK using Tailwind in #30 and @ishagupta1548 in this issue here.

How about that?

**Comment by giteshk11** [comment]
Sounds good to me 😄

**Comment by divyaYK** [comment]
Hey @ishagupta1548! 😊

Are you working on this issue? 
If not, I'd like to take it up. Please let me know!

**Comment by tiagostutz** [comment]
Hi @divyaYK ! I offered the issue to @ishagupta1548, but as she didn't give a sign that she wants to take that I'm assigning it to you and creating another issue to her.

Thanks! 🚀
